firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TakeScreenshotAndStartFeedbackActivity.java

```diff
@@ -38,7 +38,13 @@ public void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     infoText = getIntent().getCharSequenceExtra(INFO_TEXT_EXTRA_KEY);
     LogWrapper.i(TAG, "Capturing screenshot and starting feedback");
-    firebaseAppDistribution.startFeedback(infoText);
+    if (firebaseAppDistribution instanceof FirebaseAppDistributionImpl) {
+      ((FirebaseAppDistributionImpl) firebaseAppDistribution)
```

Comment:
This is gross, but I don't want to introduce a public method in the interface that takes in the feedback trigger. Any other suggestions?

Reply:
You could inject `FirebaseAppDistributionImpl` instead of `FirebaseAppDistribution`

Reply:
No, I don't think so: The code here is the (real) impl. The API (with its default no-op impl) stands on its own.

Reply:
Done.